packages/cli/commands/sandbox/sync.js

```diff
@@ -0,0 +1,298 @@
+    getEnv(accountId) === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
+  );
+
+  try {
+    logger.log('');
+    spinnies.add('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.startSync`),
+    });
+
+    // Fetches sync types based on default account. Parent account required for fetch
+    const tasks = await getAvailableSyncTypes(
+      parentAccountConfig,
+      accountConfig
+    );
+
+    initiateSyncResponse = await initiateSync(
+      parentAccountId,
+      accountId,
+      tasks,
+      accountId
+    );
+
+    uiLine();
+    logger.log(
+      i18n(`${i18nKey}.info.syncStatus`, {
+        url: `${baseUrl}/sandboxes-developer/${parentAccountId}/development`,
+      })
+    );
+    uiLine();
+
+    logger.log('');
+    spinnies.succeed('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.succeed`),
+    });
+  } catch (err) {
+    trackCommandUsage('sandbox-sync', { successful: false }, accountId);
+
+    spinnies.fail('sandboxSync', {
+      text: i18n(`${i18nKey}.loading.fail`),
+    });
+
+    logger.log('');
+    if (isMissingScopeError(err)) {
+      logger.error(
+        i18n(`${i18nKey}.failure.missingScopes`, {
+          accountName: getAccountName(parentAccountConfig),
+        })
+      );
+    } else if (
+      isSpecifiedError(
+        err,
+        429,
+        'RATE_LIMITS',
+        'sandboxes-sync-api.SYNC_IN_PROGRESS'
+      )
+    ) {
+      logger.error(
+        i18n(`${i18nKey}.failure.syncInProgress`, {
+          url: `${baseUrl}/sandboxes-developer/${parentAccountId}/syncactivitylog`,
+        })
+      );
+    } else if (
+      isSpecifiedError(
+        err,
+        403,
+        'BANNED',
+        'sandboxes-sync-api.SYNC_NOT_ALLOWED_INVALID_USERID'
+      )
+    ) {
+      // This will only trigger if a user is not a super admin of the target account.
+      logger.error(
+        i18n(`${i18nKey}.failure.notSuperAdmin`, {
+          account: getAccountName(accountConfig),
+        })
+      );
+    } else {
+      logErrorInstance(err);
+    }
+    logger.log('');
+
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  try {
+    // Handle manual exit for return key and ctrl+c
+    const onTerminate = () => {
+      logger.log('');
+      logger.log('');
+      logger.log('Exiting, sync will continue in the background.');
+      process.exit(EXIT_CODES.SUCCESS);
+    };
+    handleExit(onTerminate);
+    handleKeypress(key => {
+      if (
+        (key && key.ctrl && key.name == 'c') ||
+        key.name === 'enter' ||
+        key.name === 'return'
+      ) {
+        onTerminate();
+      }
+    });
+
+    logger.log('');
+    logger.log('Sync progress:');
+    // Poll sync task status to show progress bars
+    await pollSyncTaskStatus(parentAccountId, initiateSyncResponse.id);
+
+    logger.log('');
+    spinnies.add('syncComplete', {
+      text: i18n(`${i18nKey}.polling.syncing`),
+    });
+    spinnies.succeed('syncComplete', {
+      text: i18n(`${i18nKey}.polling.succeed`),
+    });
```

Comment:
I had to do this add + succeed here to get the "success" spinnies message to show after the poll func. The bar chart rendering was causing issues if I put the spinnies.add above the poll